datafusion/optimizer/src/common_subexpr_eliminate.rs

```diff
@@ -532,18 +545,18 @@ impl ExprMask {
 /// Go through an expression tree and generate identifier.
 ///
 /// An identifier contains information of the expression itself and its sub-expression.
-/// This visitor implementation use a stack `visit_stack` to track traversal, which
+/// This visitor implementation use a stack `f_down()` to track traversal, which
```

Comment:
I think the original was correct (the stack was called `visit_stack`) -- it seems like the stack itself is created on `f_down`

Reply:
nice catch.